component/azstorage/file_share.go

```diff
@@ -329,6 +387,20 @@ func (fs *FileShare) RenameFile(source string, target string) error {
 func (fs *FileShare) RenameDirectory(source string, target string) error {
```

Comment:
To get the rename code, in our repo run `go get github.com/Azure/azure-storage-file-go/azfile@f37a99d62e3f9ed290ff5c7b20cb184bab71367d `